packages/big-design/src/components/Timepicker/Timepicker.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef, memo, Ref } from 'react';
+
+import { getTimeIntervals } from '../../utils';
+import { Select } from '../Select';
+interface Props {
+  error?: React.ReactNode;
+  label?: string;
```

Comment:
This needs to accept both a `string` and a `FormControlLabel`. I would also make a note somewhere that the `Select` component being used provides the type guard functionality since it doesn't make sense to check it twice.

Reply:
Good catch. The `Select` type was changed recently.

Reply:
Where do you think is a good place to make that note. TimepickerPropTable?